Dockerfile

```diff
@@ -2,6 +2,9 @@ ARG version=20.04
 FROM ubuntu:$version
 ARG DEBIAN_FRONTEND=noninteractive
 
+COPY . /tmp/Homebrew
```

Comment:
Can you explain this a bit? Seems like a strange addition.

Reply:
Because the daemon copies the directory, the `linuxbrew` user will run into errors if we don't modify permissions. The previous iteration of this file copied it in just before the second `RUN` instruction, and then modified permissions at the very end (after running `brew` -- which created errors as mentioned in the issue referenced in the commit message). By copying the contents into some other path, and later moving it (in the first `RUN` instruction), we accomplish the goal while keeping root-user logic and actions entirely in the first `RUN` instruction.

The alternative here would be to copy it after the first `RUN` invocation, however, as mentioned above, permissions would be wonky -- so we'd need to move the `chown` and `chmod` command lines below this copy statement. This would introduce some slight maintenance complexity, as then we would need to work around permissions while ensuring we run the `brew` commands as a non-root user. Yes, this could be done by prefixing `su - linuxbrew -c <cmd>` for each command we need to run, however, that is clunky in and of itself and is not something I would recommend.

Reply:
Actually, I just learned that we can specify ownership with the `COPY` instruction. Neat! I'll send an updated patch.

Reply:
Thanks!